### DISCUSSION TITLE
Support for over the air updates

### DISCUSSION THREAD

**Discussion by alpap** [microsoft/devicescript#537] [Q&A] [answered]
Hi guys I would like to ask if there's going to be support for the over the air updates for the ESP32 C3 and S3 series

**Comment by pelikhan** [comment, marked as answer]
@alpap  Do you have a particular update delivery strategy in mind? We've recently added a "bundle" command in the command line to create a self-contained firmware binary with compiled devicescript and settings; that you should be able to use with an existing OAT system.

**Reply by alpap** [reply]
That is great, is there any documentation on recommended OTA and maybe examples I could follow?

**Reply by alpap** [reply]
Finally got time and worked it out you can basically use any OAT system as it imports a binary in the end

**Reply by pelikhan** [reply]
We are curious, are you allowed to share some details of your approach?

**Comment by michael-land** [comment]
@alpap  Could you please outline some steps? I'm also interested

**Comment by marcozetaa** [comment]
@alpap I also am, i really need to integrate OTA with Typescript

**Comment by inducingchaos** [comment]
@alpap Bumping this - would also like to see
